README.md

```diff
@@ -133,6 +133,12 @@ Opens a given note in your `$EDITOR`. Name can be an absolute path, or a relativ
 
 If no file-suffix is given in `note-name`, the notes will attempt to open `note-name.md` (or whatever your default suffix is set to). However, if the note-name is given an suffix, the default suffix will not be appended (e.g. `notes open note-name.txt` will open `note-name.txt`; not `note-name.md` or `note-name.txt.md`).
 
+### `notes append <note-name> [message]`
+
+Appends a given note with the test `message` from the command line. If no note yet exists, a new note of <note-name> will be created. This command also accepts stdin
+via piping. An example would be `echo "hello" | notes append <note-name>`
+Shorthand alias also available with `notes a`.
```

Comment:
Nit picking: all of this is actually a single markdown paragraph. For the rest of this file AFAICT we've just put everything on one line rather than manually wrapping with single newlines, so we might as well do that here.